lib/metamodelutil.js

```diff
@@ -89,10 +89,11 @@ function createNameTable(priorModels, metaModel) {
                 if (!findDeclaration(modelFile, type)) {
                     throw new Error(`Declaration ${type} in namespace ${namespace} not found`);
                 }
-                table[localName] = {namespace, name: type};
+                table[localName] = localName !== type ? {namespace, name: localName, resolvedName: type} : {namespace, name: type};
             });
         } else {
             (modelFile.declarations || []).forEach((decl) => {
+                // What is this used for?
```

Comment:
Why add a comment here? This comment doesn't explain the code or notify anything. Maybe just add a comment to the PR? Or raise it in Discord?

Could you remove it before you merge this?

Reply:
The code is adding types defined in the current CTO to the types table.

Reply:
Sorry I had forgot to remove the comment from the older PR.